src/plugins/intel_gpu/src/graph/program.cpp

```diff
@@ -1398,7 +1398,8 @@ void program::set_layout_optimizer_attributes(layout_optimizer& lo) {
             prim.type() != cldnn::normalize::type_id() &&
             prim.type() != cldnn::mvn::type_id() &&
             prim.type() != cldnn::gather::type_id() &&
-            prim.type() != cldnn::scatter_nd_update::type_id()) {
+            prim.type() != cldnn::scatter_nd_update::type_id() &&
+            prim.type() != cldnn::non_max_suppression::type_id()) {
```

Comment:
There is a whitelist for double blocked format too, please add there too.

Reply:
done